_maps/map_files/Whiskey_Outpost/Whiskey_Outpost_v2.dmm

```diff
@@ -6273,7 +6314,9 @@
 /turf/open/floor/plating/ground/dirt,
 /area/whiskey_outpost/outside/east)
 "Pc" = (
-/obj/machinery/door/window/westright,
+/obj/machinery/door/window/westright{
```

Comment:
aaaaaaaaaaaaaaaa @psykzz this is a dirty map edit!

Reply:
aaaaaaaaaaaaaaaaaa ill fix it in my merge conflicting pr